src/PkgServerLogAnalysis.jl

```diff
@@ -155,7 +155,64 @@ function should_ignore_parse_failure(r::RegexMatch)
     return false
 end
 
-function parse_log_line(line::AbstractString, filename::AbstractString)
+# We rewrite some regex matches
+function fetch_rewrite(m::RegexMatch, k::Symbol)
+    x = m[k]
+
+    # Rewrite timestamp into ISO8601 format
+    if k === :time_utc
+        # nginx $time_local format, e.g. [29/Sep/2016:10:20:48 +0100]
+        time_local_re = r"^\[(\d{2}\/\w{3}\/\d{4}:\d{2}:\d{2}:\d{2}) (\+|-)(\d{2})(\d{2})\]$"
+        # nginx $time_iso8601 format, e.g. [2016-09-29T10:20:48+01:00]
+        time_iso8601_re = r"^\[(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})(\+|-)(\d{2}):(\d{2})\]$"
+        if (m = match(time_local_re, x)) !== nothing
+            dt = tryparse(DateTime, m[1], dateformat"dd/uu/yyyy:HH:MM:SS")
+            offset_sign = m[2]
+            offset_hour = m[3]
+            offset_min = m[4]
+        elseif (m = match(time_iso8601_re, x)) !== nothing
+            dt = tryparse(DateTime, m[1], dateformat"yyyy-mm-ddTHH:MM:SS")
+            offset_sign = m[2]
+            offset_hour = m[3]
+            offset_min = m[4]
+        else
+            return nothing
+        end
+        dt === nothing && return nothing
+        offset = Hour(parse(Int, offset_hour)) + Minute(parse(Int, offset_min))
+        if offset_sign == "+"
+            dt -= offset
+        else # offset_sign == "-"
+            dt += offset
+        end
+        return Dates.format(dt, dateformat"yyyy-mm-ddTHH:MM:SS")
```

Comment:
Compared to before, this has the extra benefit that this column will be recognized as `Dates.DateTime` and not `String`.

Reply:
Yeah, this is nice.